docs/collectors/PhpFpmCollector.md

```diff
@@ -25,6 +34,9 @@ pm.status_path = /fpm-status
 
 Setting | Default | Description | Type
 --------|---------|-------------|-----
+uri | fpm-status | Path part of the URL, with or without the leading / | string
```

Comment:
Done. Also made lines in docstring shorter because pep8. One line is still >80 chars but it's a link, I don't know how to make it pep8-compatible without breaking the link.

It's backwards, the doc files should be automatically generated and build_doc.py doesn't work at all... Well I guess I have no right to complain if a patch isn't attached to the complaint.